Shared/Examples/ExperimentFactory.swift

```diff
@@ -19,6 +19,7 @@ class ExperimentFactory: ExampleFactory {
         table.margin = 10
         table.padding = 10
         table.showHeadersOnEveryPage = false
+        table.shouldSplitCellsOnPageBeak = false
```

Comment:
Hi Philip! 👋 I've found this repo and I'm trying to build my own PDFs using this cool library 🕵️‍♂️

Obviously I have the same issue with rendering tables, but it really seems to be fixed on this branch (at least when not using with `showHeadersOnEveryPage` set to `true`). 💪 

On the other hand it seems that you've got a typo in the property name (`beak` instead of `break`).

BTW I'd like to ask about this property, because I'm dying of curiosity. I can't find a place where you add this property to `PDFTable` in the pull request. It's also unavailable on `develop`. But it's available to use and works perfectly. Could you explain me how is it possible? Does the code is autogenerated or am I missing something? 🤔 

PS Obviously I can find commit where it's added d0fc688, but it's not listed here 🤔

Reply:
Hi @sochalewski, thanks for letting me know about the typo!

I guess I will merge the optional table splicing for now and fix the table headers in another issue.

About the property: I guess you found the right commit, but I am not entirely sure what you mean with "It's also unavailable on `develop`" as I can clearly find it
<img width="690" alt="image" src="https://user-images.githubusercontent.com/30923007/89462088-36432a80-d76d-11ea-916b-04dd09aa6a7e.png">

Did you pull the latest changes on `develop`?

Reply:
Oh, that's my fault. I looked at `PDFTable.swift` via Github web interface and didn't notice that default branch is `master`, not `develop`. Sorry for the confusion, but on the other hand the mystery is now solved 😂